### PR TITLE
Activate test runs on Python 3

### PR DESCRIPTION
**Pull request by claudep**
Just to see where we are...

### CODE REVIEW

**Comment by unho** [comment]
It seems that some tests are not passing. Is that expected?

**Comment by claudep** [comment]
Yes, sure. This pull request is not meant to be merged now, it's just a way to see the remaining work. But I think (and hope) we are rather close!

**Comment by claudep** [comment]
pip installation currently fails because python-Levenshtein==0.10.2 (Python 3 support has been added in 0.11)

**Comment by unho** [comment]
Current requirement is:

python-Levenshtein>=0.10.2,!=0.11.0

So it should be ok to use instead:

python-Levenshtein>=0.11.1

**Comment by unho** [comment]
I see that Travis now runs 7 jobs each time is fired. Perhaps all those are necessary, but please check if you can reduce the number of jobs.

**Comment by julen** [comment]
Congrats @claudep, well done :dancer:

**Comment by claudep** [comment]
@unho also did a great job at reviewing...

**Comment by unho** [comment]
lgtm. Congratulations.

**Comment by claudep** [comment]
Let's still note that with ~65% test coverage, there are probably several remaining issues with Python 3. I'll try to stay around for fixing those, at least.

**Comment by unho** [comment]
That is much appreciated.

**Comment by KevinEtienne** [comment]
Do you think this is ready to be included in the next release? It might worth adding support for `3.5` and maybe dropping `3.3`?

File: requirements/optional.txt [hunk 1]
```diff
@@ -4,8 +4,8 @@
 # Format support #
 ##################
 BeautifulSoup4>=4.3  # Trados TM
-iniparse>=0.3.1      # INI (ini2po)
-vobject>=0.6.6       # iCal (ical2po)
+iniparse>=0.3.1 ;python_version < '3.0'      # INI (ini2po)
+vobject>=0.6.6 ;python_version < '3.0'       # iCal (ical2po)
```
Comment (claudep) [inline requirements/optional.txt]: Looks like vobject could be replaced by https://pypi.python.org/pypi/python-card-me (to be tested)

Comment (unho) [inline requirements/optional.txt]: Yes, but it seems it doesn't support Python 2.6.

**Comment by unho** [comment]
@claudep It would be great if you can rebase this PR too. Also it seems there are some changes (for example "Fixed order of progress bar option names" commit) that seem to be ok to be merged to master.

**Comment by claudep** [comment]
Rebased. The remaining (random) failure is due to a header ordering issue between X-Accelerator-Marker and X-Merge-On. They are not in `poheader.header_order` so their ordering is undefined.

**Comment by unho** [comment]
@claudep Thanks. I would also merge the "Upgraded lxml minimal requirement for Python 3" commit as it won't harm.

**Comment by unho** [comment]
@claudep Just merged some fix for random order in 2fa69bb38 and followup commit. Would be great if you can rebase again and merge the "Upgraded lxml minimal requirement for Python 3" commit.

**Comment by unho** [comment]
This was merge to master. The support for INI and iCalendar is disabled when using Python 3.

**Comment by claudep** [comment]
\o/

**Pull request closed without merging**